unit_tests/server/test_async_disconnect_handler_main.cpp

```diff
@@ -0,0 +1,98 @@
+/*
+ *
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+#define CATCH_CONFIG_MAIN
+#include "catch2/catch.hpp"
+
+#include "async_disconnect_handler.hpp"
+
+#include <thread>
+#include <random>
+#include <chrono>
+#include <atomic>
+
```

Comment:
```suggestion
#include <array>

```
I think `<array>` is needed to fix window build error

Reply:
Thank you. I missed it.